Added function to get holder length

Feuerbowle opened the pull request:
Added the function to get the complete length of the holder

Feuerbowle committed:
Added function to get holder length

Added the function to get the complete length of the holder

Feuerbowle committed:
Added function to get gauge length

Added the function to get the gauge length of the tool

lukeedw commented on Delcam.ProductInterface.PowerMILL.Test/PMEntityTests/PMToolTest.cs:
```diff
@@ -513,6 +513,28 @@ public void HolderOverHangTest()
+                Assert.Inconclusive("Test not available for this version of PowerMILL");
+            }
+            var tool = _powerMill.ActiveProject.Tools[12];
+            Assert.AreEqual((MM)50.0, tool.HolderLength());
```
Can you use Assert.That((MM)50.0, Is.EqualTo(tool.HolderLength());  This is the preferred method of doing it.

lukeedw commented on Delcam.ProductInterface.PowerMILL.Test/PMEntityTests/PMToolTest.cs:
```diff
@@ -513,6 +513,28 @@ public void HolderOverHangTest()
+                Assert.Inconclusive("Test not available for this version of PowerMILL");
+            }
+            var tool = _powerMill.ActiveProject.Tools[12];
+            Assert.AreEqual((MM)95.0, tool.GaugeLength());
```
Can you use Assert.That((MM)95.0, Is.EqualTo(tool.GaugeLength());  This is the preferred method of doing it.

lukeedw commented on Delcam.ProductInterface.PowerMILL/Classes/PMTool.cs:
```diff
@@ -391,6 +391,37 @@ public MM HolderElementLength(int index)
+        /// <summary>
+        /// Gets the length of the complete Holder
+        /// </summary>
+        public MM HolderLength()
```
Can you make this a readonly property instead?

lukeedw commented on Delcam.ProductInterface.PowerMILL/Classes/PMTool.cs:
```diff
@@ -391,6 +391,37 @@ public MM HolderElementLength(int index)
+        /// <summary>
+        /// Gets the gauge length of the tool
+        /// </summary>
+        public MM GaugeLength()
```
Can you make this a readonly property instead?

lukeedw reviewed (changes requested):
Great!  Just a few minor changes required to keep consistency.

Feuerbowle committed:
Requested changes

- Changed the testing method to 'Is.EqualTo() for HolderLength and GaugeLength
- HolderLength and Gaugelength are now a readonly property
